.env.example

```diff
@@ -29,6 +29,10 @@ NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/admin/dashboard #leave as-is
 ##sentry
 NEXT_PUBLIC_SENTRY_DSN=<SENTRY_DSN> #only necessary if working on error tracking, otherwise leave as-is
 
+##inngest
```

Comment:
Can you add these to the `env.ts` file as well so we get intellisense when working with those variables?